### ISSUE TITLE
Unsupported language warning on Mac M1

### ISSUE THREAD

**Issue by SL-23** [TypeStrong/typedoc#2764]
## Search terms

[warning] Options specified "en" as the language to use, but TypeDoc does not support it.

## Expected Behavior

No warning with below code

``` 
const app = await Typedoc.Application.bootstrapWithPlugins({
    entryPoints: [],
  });
  const converted = (await app).convert();
  if (await converted) {
    console.log("okk");
  }

```

## Actual Behavior

<img width="566" alt="Screenshot 2024-11-04 at 9 36 40 PM" src="https://github.com/user-attachments/assets/be3f4dff-0f84-4307-9abb-4099b52d9e42">

## Steps to reproduce the bug

See [my testing app](https://github.com/SL-23/next-style-x)

To reproduce:
1. Go to my testing app
2. Clone my app
3. `npm install`
4. `nvm use v18.18.2`
5. `npm run gen-doc`

## Environment

-   Typedoc version: every version since `v0.26.0`
-   TypeScript version: `^5`
-   Node.js version: `v18.18.2`
-   OS: Mac M1 chip (M2 chip has the same issue, but not i9)

**Comment by Gerrit0** [comment]
This is happening because of the check for `ts-node` in the internationalization class. TypeDoc tries to load a TS version of the locales as that's what it should use in development.

This should already be fixed in 0.27 as TypeDoc has moved away from ts-node

**Comment by SL-23** [comment]
Thanks @Gerrit0, will close my PR then.
